docs/stdlib/datetime.rst

```diff
@@ -905,6 +903,48 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         ...   <duration>'30 hours', 'totalseconds');
         {108000}
 
+    **Gotchas**
```

Comment:
Can you try using `.. rubric:: Gotchas`; would it look good visually? Semantically it would still be a header, not a bold text, which is better in this case.

Reply:
I'll give that a shot. Thank you!

Reply:
That seemed to work well, @1st1. Thanks for the suggestion!